scripts/approvalYears.py

```diff
@@ -70,7 +73,7 @@ def getMainDir():
     return curr
 
 def resolveName(name): # returns empty string if resolver can't do anything with this name
-    if name in resolverCache.keys():
+    if name in list(resolverCache.keys()):
```

Comment:
Should just be "if name in resolverCache:" (unless I'm missing something)

Reply:
Python3 versus python2 issue. 2to3 automagically has made this change as .keys() no longer returns a list in python3